particles/TestingExample/ComputePeopleStats.kt

```diff
@@ -0,0 +1,25 @@
+package arcs.unit.test.example
+
+import arcs.core.storage.api.Handle
+
+/**
+ * Particle computing a median age of input people - an example for unit testing.
+ */
+class ComputePeopleStats : AbstractComputePeopleStats() {
+
+    override suspend fun onHandleUpdate(handle: Handle) {
```

Comment:
Out of curiosity, do you happen to know if it's possible to use assert functions (e.g. assertThat) inside particle code? It might be useful for us to be able to write test particles that exercise certain parts of the Arcs SDK, and assert various things in them directly. Not for now, or part of this PR, but I was just curious.

Reply:
It should work... now, the question is would the thrown error be propagated to the test or be swallowed? Things will get much simpler when we get rid of suspendable handle functions, as then we won't need to run tests with coroutines - then it should propagate naturally.